diff --git a/internal/libdocker/container.go b/internal/libdocker/container.go
--- a/internal/libdocker/container.go
+++ b/internal/libdocker/container.go
@@ -96,6 +96,10 @@ func (b *ContainerBackend) CreateContainer(ctx context.Context, imageName string
 	createOpts := docker.CreateContainerOptions{
 		Context: ctx,
 		Name:    opt.Name,
+		HostConfig: &docker.HostConfig{Sysctls: map[string]string{
+			"net.ipv4.ip_local_port_range": "1024 65535",
+			"net.ipv4.tcp_tw_reuse":        "1",
+		}},
 		Config: &docker.Config{
 			Image:  imageName,
 			Env:    vars,
